ec2removeimg

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python3
+#!/home/kberger/venv/ec2imgutils/bin/python3
```

Comment:
Certainly the interpreter does not live in your home directory on every system ;)